ajax/libs/microplugin/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "microplugin",
+  "keywords": [
+    "extensibility",
+    "plugins",
+    "architecture",
+    "require",
+    "dependencies"
+  ],
+  "description": "A lightweight plugin / dependency system for javascript libraries.",
+  "version": "0.0.3",
+  "author": "Brian Reavis <brian@thirdroute.com>",
+  "main": "src/microplugin.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/brianreavis/microplugin.js.git"
+  },
+  "dependencies": {},
+  "engines": {
+    "node": "*"
+  },
+  "npmName": "microplugin",
+  "npmFileMap": [
+    {
+      "basePath": "/src",
+      "files": [
+        "**/*"
```

Comment:
we only use "**/*" in `dist` or `build` folder, please use `microplugin*` here, thanks